frameit/lib/frameit/editor.rb

```diff
@@ -484,12 +491,6 @@ def fetch_text(type)
 
       # No string files, fallback to Framefile config
       text = fetch_config[type.to_s]['text'] if fetch_config[type.to_s] && fetch_config[type.to_s]['text'] && fetch_config[type.to_s]['text'].length > 0 # Ignore empty string
-
-      if type == :title && !text
-        # title is mandatory
-        UI.user_error!("Could not get title for screenshot #{screenshot.path}. Please provide one in your Framefile.json or title.strings")
```

Comment:
Doesn't `user_error!` throw an exception? Wouldn't that mean that before your change this stopped if `title` was missing, and now those screenshots are just skipped?

Reply:
It does throw, but exceptions here are being caught and swallowed within a loop.  See https://github.com/fastlane/fastlane/blob/22f845f82317933635044d55d13d146c8afae431/frameit/lib/frameit/runner.rb#L47

Reply:
Oh right, and then converted to a normal `UI.error` that is output (with its backtrace). So the only functional change here is that before it was a `UI.error` and now is a `UI.message`, correct?

Reply:
Correct! And, as you mentioned, no longer printing the backtrace on the `UI.message`